releases/v21.2.1.md

```diff
@@ -0,0 +1,55 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.2.1.src.tgz"><button id="source" class="filter-button" data-scope="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include_cached windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v21.2.1
+~~~
+
+### Bug fixes
+
+- The timeout check of Raft application during upgrade migrations has been increased from 5 seconds to 1 minute and is now controllable via the [cluster setting](../v21.2/cluster-settings.html) `kv.migration.migrate_application.timeout`. This makes migrations less likely to fail in clusters with ongoing rebalancing activity during upgrade migrations. [#73061][#73061]
+- Fixed a bug where [`BACKUP ... with revision_history`](../v21.2/take-backups-with-revision-history-and-restore-from-a-point-in-time.html) would fail on an upgraded, but un-finalized cluster. It will now succeed. [#73050][#73050]
+- Fixed a bug that could cause some semi [lookup joins](../v21.2/joins.html#lookup-joins) on [`REGIONAL BY ROW`](../v21.2/multiregion-overview.html#regional-by-row-tables) tables to return early before finding all data. This bug is currently only present in [v21.2.0](v21.2.0.html). This problem only manifested if there was an `ON` condition on top of the equality condition used for the lookup join, and the lookup columns did not form a key in the index being looked up. The primary impact of this issue for most users relates to uniqueness checks on mutations of `REGIONAL BY ROW` tables, since uniqueness checks are implemented with a semi lookup join with an `ON` condition. The result of this bug was that uniqueness checks were not comprehensive, and could miss an existing duplicate key on a remote node. This could cause data to be erroneously inserted with a duplicate key when it should have failed the uniqueness check. [#73063][#73063]
+- [Backups](../v21.2/take-full-and-incremental-backups.html) taken while a cluster contains a mix of v21.2 and v21.1 nodes may fail. [Upgrading](../v21.2/upgrade-cockroach-version.html) the entire cluster to v21.2 should resolve the issues. The [technical advisory 72839](../advisories/a72839.html) provides more information about possible [remediations](../advisories/a72839.html#mitigation). The error returned after a backup failure in this case now also directs the user to the technical advisory. [#72880][#72880]
+- Fixed a bug that caused a [full-cluster backup](../v21.2/backup.html#backup-a-cluster) to fail while upgrading from v21.1 to v21.2. This caused an error, because the `system.tenant_usage` table, which is present in v21.2, is not present in v21.1. [#72840][#72840]
+- Fixed a bug where cluster backups were backing up opt-out system tables unexpectedly. [#71368][#71368]
+
+### Contributors
+
+This release includes 0 merged PRs by 6 authors.
```

Comment:
@kathancox Is `0` merged PRs correct?

Reply:
Ahh yes, missed this. Looks like this is as a result of commits being cherry-picked for this particular release. I will make a change after @rmloveland's review. Thanks!